docs/about/change_log.rst

```diff
@@ -1,92 +1,109 @@
+Version 0.1.0
+-------------
 
-Version `0.1.0` is the initial development release of PlasmaPy.  This
+Version 0.1.0 is the initial development release of PlasmaPy.  This
 version is a prototype and a preview, and is not feature complete.
 Significant changes to the API are expected to occur between versions
-`0.1.0` and `0.2.0`.
+0.1.0 and 0.2.0, including backward incompatible changes.
 
 .. _change-log-0.1.0-new:
 
 New Features
 ~~~~~~~~~~~~
 
-* The vision statement describes the motivation of early PlasmaPy
-  developers in creating a fully open source Python package for plasma
-  physics.
+* Composed :ref:`plasmapy-vision-statement`.
 
-* Added a code of conduct.
+* Adopted the :ref:`plasmapy-code-of-conduct`.
 
-* Added a contribution guide.
+* Created a guide on :ref:`contributing-to-plasmapy`.
 
-* Adopted a BSD 3-clause license and added protections against
-  software patents.
+* Adopted a permissive BSD 3-clause `license
+  <https://github.com/PlasmaPy/PlasmaPy/blob/master/LICENSE.md>`_ with
+  protections against software patents.
 
-* Set up continuous integration testing with Travis CI, CircleCI, and
-  AppVeyor, along with test coverage checks with Coveralls.
+* Set up continuous integration testing with `Travis CI
+  <https://travis-ci.org/>`_, `CircleCI <https://circleci.com/>`_, and
+  `AppVeyor <https://www.appveyor.com/>`_, along with test coverage
+  checks with `Coveralls <https://coveralls.io/>`_.
 
 * Decided upon code and docstring style conventions and set up
-  automated code style checks with pep8speaks.
+  automated code style checks with `pep8speaks
+  <https://pep8speaks.com/>`_.
 
-* Set up automated documentation builds with Sphinx that are hosted on
-  Read the Docs.
+* Developed `online documentation for PlasmaPy
+  <http://docs.plasmapy.org>`_ that is hosted by `Read the Docs
+  <https://readthedocs.org/>`_.
 
-* Adopted use of `~astropy.units` as a units package.  Allowed import of
-  units as ``import plasmapy.units as u``.
+  - Automated documentation builds with `Sphinx
+    <http://www.sphinx-doc.org/>`_.
 
-* Created the `~plasmapy.atomic` subpackage to provide easy access to commonly
-  used atomic data.
+  - Wrote narrative documentation for each subpackage.
 
-  - Created functional interface to access particle properties and find
-    the energy released from nuclear reactions.
+* Adopted use of `~astropy.units` as a units package.
 
-  - Created `~plasmapy.atomic.Particle` class.
+* Created the `~plasmapy.atomic` subpackage to provide easy access to
+  commonly used atomic data.
 
-  - Created `~particle_input` decorator.
+  - Created a functional interface to access particle properties and
+    find the energy released from nuclear reactions.
 
-* Created `~plasmapy.classes` subpackage with the
-  `~plasmapy.classes.Plasma` and `~plasmapy.classes.Species` classes.
+  - Created the `~plasmapy.atomic.Particle` class as an object-oriented
+    interface to the `~plasmapy.atomic` subpackage.
```

Comment:
Some of these get linked to stuff in the docs here work and some don't. Here, atomic doesn't link and Particle does. I wouldn't worry about it for now, much, I just wanted to state a point of my confusion.

Actually, maybe that's because the docs for atomic is custom, with [source here](https://1395-46810954-gh.circle-artifacts.com/0/home/circleci/project/docs/_build/html/_sources/atomic/index.rst.txt), while `Particle` is generated by automodapi [as seen here](https://1395-46810954-gh.circle-artifacts.com/0/home/circleci/project/docs/_build/html/_sources/api/plasmapy.atomic.Particle.rst.txt) and does something right that we aren't aware we should be doing?

I'm looking at the page source for Atomic and this line seems like it could  be a culprit: `.. py:module:: atomic`

In [Sunpy docs](http://docs.sunpy.org/en/stable/py-modindex.html), each item under the module index sits right there under `sympy` in the namespace. This would kinda seem to imply we should have those lines as `.. py:module:: plasmapy.atomic`. Would you like me to go through these and update?

Oh look, I'm worrying about it now. :laughing:

Reply:
>  Would you like me to go through these and update?

If you have a chance, please do!